types/handsontable/index.d.ts

```diff
@@ -494,19 +494,16 @@ declare namespace ht {
 }
 
 interface Handsontable {
-    new (element: Element, options: ht.Options): ht.Methods;
-    plugins: ht.Plugins;
-    hooks: ht.Hooks;
-    Dom: ht.Dom;
-    dom: ht.Dom;
-    helper: ht.Helper;
-    utils: ht.Utils;
-    renderers: ht.Renderers;
-    editors: ht.Editors;
+    new (element: Element, options: Handsontable.Options): Handsontable.Methods;
+    plugins: Handsontable.Plugins;
+    hooks: Handsontable.Hooks;
+    Dom: Handsontable.Dom;
+    dom: Handsontable.Dom;
+    helper: Handsontable.Helper;
+    utils: Handsontable.Utils;
+    renderers: Handsontable.Renderers;
+    editors: Handsontable.Editors;
```

Comment:
I've made this change and will push once I get an internet connection. In order to get it to build, I had to introduce the module declaration again.